_commands/basics/gcc.md

```diff
@@ -0,0 +1,106 @@
+---
+---
+
+gcc flags
+--
+
+`gcc` is a compiler command that has the ability to execute options or 'Flags' when compiling.  These flags can be grouped and executed via the command line:
+    '-o' places output in a file.
+    '-g' turns on debugging information.
+    '-Werror' makes all warnings into errors.
+    '-Wall' enables all the warnings.
+     '-ar' creates an archive instead of a program.
+
+~~~ bash
+$ gcc test.c -o test
+$ gcc -g test.c
+$ gcc -Werror test.c
+$ gcc -Wall test.c
+$ gcc -ar test.c
+~~~
+
+
+
+
+
+
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `-o`
+~~~ bash
+$ gcc test.c -o test
+~~~
+
+##### Break it down
+~~~ bash
+*  This is a very simple example where test.c is being compiled and outputted to the test file.  The output filename is specified after the flag.  Input: test.c, Output: test
```

Comment:
hi Matt, I am not sure if I am suppose to do something about this?  Could you please, advise?